src/main/definitions/case.ts

```diff
@@ -478,6 +488,12 @@ export const enum HearingPreference {
   NEITHER = 'Neither',
 }
 
+export const enum HearingPreferenceET3 {
+  VIDEO = 'Video hearings',
+  PHONE = 'Phone hearings',
+  NEITHER = 'No, I cannot take part in either a video or phone hearing',
```

Comment:
It probably won't cause an issue but I guess this line can be removed as we are no longer capturing it
```suggestion
```